server/common.js

```diff
@@ -41,4 +42,18 @@ const userTypes = Object.freeze({
     ADMIN     : 0,
     STUDENT   : 1
 });
-exports.userTypes = userTypes;
\ No newline at end of file
+exports.userTypes = userTypes;
+
+var randomize = function(data) {
```

Comment:
ramdomizeList, incase we want other randomization functions

Reply:
good idea